### ISSUE TITLE
Possible to upload videos?

### ISSUE THREAD

**Issue by stursby** [lukecyca/python-instagram-upload#7]
I grabbed a .mp4 right off Instagram.com so I know the file type is right, but the upload isn't working... any thoughts on how to get the script working for video?

**Comment by torlakfaruk** [comment]
@stursby did you solve the problem? I'm searching a way for it too.

**Comment by stursby** [comment]
@torlakfaruk no, I didn't. I messed around with different video types, encoding, etc... couldn't get it to work.
